chemie/events/models.py

```diff
@@ -313,6 +313,12 @@ class SocialEventRegistration(BaseRegistration):
         null=True,
         blank=True,
     )
+    registration_group_members = models.ManyToManyField(
+        User,
+        blank=True,
+        verbose_name=" ",
```

Comment:
The verbose name is set to blank to disable the verbose name from being displayed in the template. Might be a better way to do this but idc =)

Reply:
Could be done by setting the label to an empty string in the ModelForm instead:
`fields = {"registration_group_members": ""}` as done [here](https://stackoverflow.com/a/36905089)

Reply:
Thanks!